modules/update-service-delete-service-web-console.adoc

```diff
@@ -0,0 +1,18 @@
+[id="update-service-delete-service-web-console_{context}"]
+= Deleting an OpenShift Update Service application by using the web console
+
+You can use the {product-title} web console to delete an OpenShift Update Service application by using the OpenShift Update Service Operator.
+
+.Prerequisites
+
+* The OpenShift Update Service Operator has already been installed.
+
+.Procedure
+
+To delete an OpenShift Update Service application by using the {product-title} web console:
+
+. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+
```

Comment:
Need to insert the step `Choose OpenShift Update Service from the list of available Operators` .

Reply:
Fixed